### PR TITLE
Document GitOps Release Notes for 1.6.7

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**Aligned team**: Dev Tools

**Purpose**: To resolve the following issues:
https://issues.redhat.com/browse/RHDEVDOCS-5218

**OCP version this PR applies to**: enterprise-`4.9`

**Link to docs preview**: 
- [Release notes for Red Hat OpenShift GitOps 1.6.7](https://59016--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html#gitops-release-notes-1-6-7_gitops-release-notes)

**SME review**: @reginapizza, @iam-veeramalla 
**QE review**: @varshab1210 
**Peer-review**:

### CODE REVIEW

File: modules/gitops-release-notes-1-6-7.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+[id="gitops-release-notes-1-6-7_{context}"]
+= Release notes for {gitops-title} 1.6.7
+
+{gitops-title} 1.6.7 is now available on {product-title} 4.8, 4.9, 4.10, and 4.11.
+
+[id="fixed-issues-1-6-7_{context}"]
+== Fixed issues
+The following issue has been resolved in the current release:
+
+* Before this update, all versions of the Argo CD Operator, starting with v0.5.0 were vulnerable to an information disclosure flaw. As a result, unauthorized users could enumerate application names by inspecting API error messages and use the discovered application names as the starting point of another attack. For example, the attacker might use their knowledge of an application name to convince an administrator to grant higher privileges. This update fixes the CVE-2022-41354 error. link:https://issues.redhat.com/browse/GITOPS-2635[GITOPS-2635], link:https://access.redhat.com/security/cve/CVE-2022-41354[CVE-2022-41354]
```
Comment (reginapizza) [inline modules/gitops-release-notes-1-6-7.adoc:16]: https://issues.redhat.com/browse/GITOPS-2635 shows me that I either don't have permission to view it or it's been deleted :/

**Review by reginaelyse — CHANGES_REQUESTED** [review]

**Comment by Srivaralakshmi** [comment]
**SME review:** @reginapizza, @iam-veeramalla
**QE review:** @varshab1210 , @ciiay

PTAL at the following PRs:

* https://github.com/openshift/openshift-docs/pull/59016 - For `4.11`
* https://github.com/openshift/openshift-docs/pull/59018 - For `4.10`
* https://github.com/openshift/openshift-docs/pull/59019 - For `4.8`
* https://github.com/openshift/openshift-docs/pull/59023 - For `4.9`

The content for 1.6.7 is the same in the PRs. I have raised multiple PRs to avoid merge conflicts as the content supported is different for all OCP versions.

This will be useful during the PR merge. Thanks for your understanding and cooperation.

**Review by reginaelyse — APPROVED** [review]

**Review by rolfedh — APPROVED** [review]
Peer review: LGTM

**Comment by varshab1210** [comment]
LGTM, thanks

**Pull request merged**